doc/articles/get-started-vsmac.md

```diff
@@ -3,34 +3,52 @@
 While it is easier to create apps using Uno on Windows, you can also create all but UWP apps on your Mac.
 
 ## Prerequisites
-* [**Visual Studio for Mac**](https://visualstudio.microsoft.com/vs/mac/)
+* [**Visual Studio for Mac 8.8**](https://visualstudio.microsoft.com/vs/mac/)
 * [**Xcode**](https://apps.apple.com/us/app/xcode/id497799835?mt=12) 10.0 or higher
 * An [**Apple ID**](https://support.apple.com/en-us/HT204316)
-* .NET Core 3.1
-
-## Modifying Existing Uno App
-
-1. Open project in Visual Studio for Mac
- ![new-project](Assets/quick-start/vs-mac-open-project.png)
-Once open, you should see your folder structure set up like this:
+* [.NET Core SDK 3.1](https://dotnet.microsoft.com/download/dotnet-core/thank-you/sdk-3.1.404-macos-x64-installer)
+* [**GTK+3**](https://formulae.brew.sh/formula/gtk+3) for running the Skia/GTK projects
+
+## Installing the dotnet new templates
+In order to create a new Uno Project, you'll need to open a terminal and run the following:
```

Comment:
Link to the complete `dotnet new` doc? https://github.com/unoplatform/uno/blob/master/doc/articles/get-started-dotnet-new.md